cmd/plugins_test.go

```diff
@@ -28,24 +31,59 @@ import (
 var initChan = make(chan struct{}, 256)
 var testDirRoot string
 
+const (
+	rootDir   string = "./"
+	prefixDir string = "plugin_test_tempdir"
+)
+
 // makeDirWithSharedObjects creates a new temporary directory containing files under the runtime directory
 // it compiles all .go files into shared object files with extension ext in the corresponding directory
 // It returns the root of the directory and a cleanup function.
-func makeDirWithSharedObjects(files map[string]string, ext string) (root string, cleanup func()) {
-	root, cleanup, err := test.MakeTempFS("./", "plugin_test_tempdir", files)
+//
+// Code is duplicated from MakeTempFS() due to https://github.com/open-policy-agent/opa/issues/1185
+func makeDirWithSharedObjects(files map[string]string, ext string) (tempRootDir string, cleanup func()) {
+
+	tempRootDir, err := ioutil.TempDir(rootDir, prefixDir)
 	if err != nil {
 		panic(err)
 	}
+	cleanup = func() {
+		if err := os.RemoveAll(tempRootDir); err != nil {
+			fmt.Println("Failed to cleanup directory: ", tempRootDir)
+		}
+	}
+	// We install the signal handler soon after the creation of the temp directory
+	signalHandler(tempRootDir, cleanup)
+
+	for path, content := range files {
+		dirname, filename := filepath.Split(path)
+		dirPath := filepath.Join(tempRootDir, dirname)
+		fmt.Println("dirpath", dirPath)
+		if err := os.MkdirAll(dirPath, 0777); err != nil {
+			fmt.Println("err", err)
+			panic(err)
```

Comment:
💭 there's quite a few `panic()`s in here -- would it make sense to add an `error` return instead? What do you think? 😃 
Alternatively, we could pass in `*testing.T` and call `t.Fatal`. 🤔

Reply:
This is existing code. When fixing a bug try to leave in place anything out of scope. We can open another issue to fix these.

Reply:
Yup fair enough. Sorry I hadn't checked that 😃